scripts/appveyor-tool.ps1

```diff
@@ -129,6 +129,7 @@ Function InstallRtools {
     $gcc_path = $env:GCC_PATH
   }
   $env:PATH = $RtoolsDrive + '\Rtools\bin;' + $RtoolsDrive + '\Rtools\MinGW\bin;' + $RtoolsDrive + '\Rtools\' + $gcc_path + '\bin;' + $env:PATH
+  $env:BINPREF=$RtoolsDrive + '/Rtools/mingw_$(WIN)/bin/'
```

Comment:
I'd really like to fix "devel" builds, I'd also like to understand what's happening here. Could you point me at documentation?

Which of your build actually creates an archive with both 32-bit and 64-bit versions? Are the tests also run twice? Do we need to add Rtools\mingw_??\bin to PATH anyway?